src/main/java/nl/tudelft/model/wall/MovingWall.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.model.wall;
+
+import nl.tudelft.semgroup4.Modifiable;
+
+import org.newdawn.slick.Image;
+import org.newdawn.slick.SlickException;
+
+public abstract class MovingWall extends AbstractWall {
```

Comment:
PMD asks for this abstract class to be called AbstractMovingWall. Could you change the name?

< naming